CenterForOpenScience/osf.io
# [#OSF-5809]Failed registrations with 100+ nodes

## Purpose
Fix the problem that projects with 100 + nodes registration will fail.


## QA notes:
Before change:
Steps to Reproduce:
1.On staging, go to https://staging.osf.io/yxzrn/
2.Create a fork of this project
3.Or make your own with 100+ nodes
4.Go to the registrations tab of the 100+ node project
5.Make a new registration
6.See that it fails
After change:
6. the registration should success



## Side effects




## Ticket
https://openscience.atlassian.net/browse/OSF-5809
